dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -47,7 +47,8 @@ winrt::hstring NavigationViewItemAutomationPeer::GetNameCore()
 winrt::IInspectable NavigationViewItemAutomationPeer::GetPatternCore(winrt::PatternInterface const& pattern)
 {
     if (pattern == winrt::PatternInterface::SelectionItem ||
-        pattern == winrt::PatternInterface::Invoke ||
+        // Only provide Invoke pattern when we are a ListItem i.e. not on top navigation
```

Comment:
> not on top navigation [](start = 66, length = 22)

From the issue comments, it sounds like we don't want to support invoke on top or left navigation.

Reply:
Hmm I see. So should NavigationView be "Tab" control in left and top mode then too?

Reply:
@YuliKl What do you think?

Reply:
From what I read, Invoke and SelectionItem patterns are supposed to be mutually exclusive. And we need nav view items to support the SelectionItem pattern, so I believe that means removing the Invoke pattern regardless of NavigationView's display mode.

Reply:
I see, thanks for the clarification. This PR is inline with that requirement in its current state.